README.md

```diff
@@ -367,22 +367,74 @@ Address Family Identifier (AFI). Required. Valid values are `ipv4` and `ipv6`.
 ##### `safi`
 Sub Address Family Identifier (SAFI). Required. Valid values are `unicast` and `multicast`.
 
+##### `additional_paths_install`
+install a backup path into the forwarding table and provide prefix
```

Comment:
fyi, you don't have to wrap text lines in markdown files. In fact I'd prefer that you don't since it makes future edits harder.